.github/workflows/deploy.yaml

```
@@ -0,0 +1,37 @@
+name: Deploy
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    types: [opened, synchronize, reopened]
```

I don't think we want CD, given we have no tests. I would be inclined to require manual deploy

```suggestion
  workflow_dispatch:
```

---

True